app/services/create_new_device_alert.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class CreateNewDeviceAlert < ApplicationJob
+  include UserAlerts
```

Comment:
What is being included here?

Reply:
That includes the method: `AlertUserAboutNewDevice.send_alert(user)`

Reply:
Where is `UserAlerts` defined?